packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -146,8 +150,7 @@ export function getMomentTemplateBanner(
 
                             <section css={styles.ctasContainer}>
                                 <MomentTemplateBannerCtas
-                                    mainContent={content.mainContent}
-                                    mobileContent={content.mobileContent}
+                                    mainOrMobileContent={content[mainOrMobileContent]}
```

Comment:
very neat, removes half the duplicated components between the MomentTemplateBannerCtas  <hide>'s